net-wireless/blueman/blueman-9999.ebuild

```diff
@@ -67,14 +67,11 @@ RDEPEND="
 			>=net-misc/networkmanager-0.8[introspection]
 		)
 	)
-	policykit? (
```

Comment:
How dare you change my perfect coding style?!

Reply:
I try to leave a personal note in every code snippet I touch.